team.md

```diff
@@ -13,6 +13,10 @@ topLevel: true
 - Geoffrey Boushey (@boushey), Applications Programmer
 - Dave Leonard (@dartajax), User Support
 
+### Emeritus
```

Comment:
We probably don't want to actually create a segment like this, rather just keep the separate blocks of names as they are now, and add a new block at the bottom made of Loki & Rolf.